en/asgardeo/docs/apis/organization-apis/restapis/notification-senders.yaml

```diff
@@ -0,0 +1,1363 @@
+        "401":
+          description: Unauthorized
+        "403":
+          description: Forbidden
+        "404":
+          description: Not Found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        "405":
+          description: Method Not Allowed.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        "500":
+          description: Server Error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+      x-codeSamples:
+        - lang: Curl
+          source: |
+            curl -X 'DELETE' \
+            'https://api.asgardeo.io/t/{root-organization-name}/o/api/server/v1/notification-senders/email/{email-sender-name}' \
+            -H 'accept: */*' \
+            -H 'Authorization: Bearer {bearer_token}'                  
+  /notification-senders/sms:
+    get:
+      tags:
+        - SMS Senders
+      summary: Get a list of SMS senders
+      description: |
+        This API provides the capability to retrieve a list of SMS notification senders.<br>
+        
+        <b>Scope(Permission) required:</b> `internal_notification_senders_view`
+      operationId: getSMSSenders
+      responses:
+        "200":
+          description: Successful response
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/SMSProviderList'
+        "400":
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        "401":
+          description: Unauthorized
+        "403":
+          description: Forbidden
+        "404":
+          description: Not Found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        "405":
+          description: Method Not Allowed.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        "500":
+          description: Server Error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+      x-codeSamples:
+        - lang: Curl
+          source: |
+            curl -X 'GET' \
+            'https://api.asgardeo.io/t/{root-organization-name}/o/api/server/v1/notification-senders/sms' \
+            -H 'accept: application/json' \
+            -H 'Authorization: Bearer {bearer_token}'                  
+    post:
+      tags:
+        - SMS Senders
+      summary: Create a SMS sender
+      description: |
+        This API provides the capability to create a SMS sender.
+        If the 'name' is not not defined, 'SMSPublisher' is taken as the default name.<br>
```

Comment:
```suggestion
        If the 'name' is not defined, 'SMSPublisher' is taken as the default name.<br>
```

Reply:
Fixed with https://github.com/wso2/docs-is/pull/5161/commits/d8a6c9c96654a492b94e6d7fb49f9243c440eafd